test/components/flatcar-linux-update-operator/fluo_reboot_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet_fluo
+// +build disruptivee2e
+
+package fluo //nolint:testpackage
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+func TestReboot(t *testing.T) {
+	t.Parallel()
+
+	client := testutil.CreateKubeClient(t)
+
+	// Select a node from the general worker pool.
+	nodesList, err := client.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{
+		LabelSelector: "fluo-test-pool=true",
+	})
+	if err != nil {
+		t.Fatalf("Listing nodes with label 'fluo-test-pool=true': %v", err)
+	}
+
+	nodes := nodesList.Items
+	if len(nodes) < 1 {
+		t.Fatalf("Wanted one or more nodes found none.")
+	}
+
+	// Select a node to remove annotation
+	chosenNode := nodes[0]
+	t.Logf("Chosen node to reboot: %s", chosenNode.Name)
+
+	// Remove annotation.
+	delete(chosenNode.Annotations, "flatcar-linux-update.v1.flatcar-linux.net/reboot-paused")
```

Comment:
This assumes the annotation is in place. Can we just not set it when we run `packet_fluo` pipeline?

Or should we rather say in the comment that we ensure that the annotation is not set?

Reply:
I don't want to reboot all the nodes in the pipeline randomly. So I will let it happen in the controlled way. I will just add a comment.